src/client/components/Terms.tsx

```diff
@@ -4,7 +4,7 @@ import { space, textSans } from '@guardian/source-foundations';
 import { ExternalLink } from '@/client/components/ExternalLink';
 
 export const termsContainer = css`
-  margin-top: ${space[5]}px;]};
+  margin-top: ${space[5]}px;
```

Comment:
Prettier error. I thought this was fixed in main? I don't see any issue on the accual sign in page though.